### ISSUE TITLE
🐞 Issues with Tally list and Tally ceremony status

### ISSUE THREAD

**Issue by edulix** [sequentech/meta#1506]
### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I have attached screenshots and/or the specific steps to reproduce this issue.

### Current Behavior

Tally list don't seem to refresh.

https://github.com/user-attachments/assets/f3079bea-8b19-49d6-80f2-0dd71d26a69f

### Screenshots

Now trustees see only the green key action button in STARTED and CONNECTED tally session status:
<img width="1157" alt="image" src="https://github.com/user-attachments/assets/df4b7239-2060-4621-a1dd-31452a9c413e">

Also, Just after cancelling a tally, it gets refreshed (and it allows to cancel in CONNECTED status):

https://github.com/user-attachments/assets/9afc62b7-39b8-409d-8a00-d100125adecb

And after cancelling both trustees and admins only see the status screen:
<img width="1157" alt="image" src="https://github.com/user-attachments/assets/9c07096b-11c0-44d4-9624-68c1e6bf7b2d">

### Expected Behavior

Should refresh. In this, and other cases, like if someone else from another user session or tab creates a new tally session etc.

### Tasks

```[tasklist]
# Tasks
- [x] Issue has been triaged
- [x] Successful replication of the defect
- [ ] Successful defect resolution merged
```

```[tasklist]
# Main PRs
- [ ] https://github.com/sequentech/step/pull/595
```

**Comment by yvwbimpong** [comment]
https://github.com/user-attachments/assets/e08b3487-853c-4061-a844-edd72c11d0c3

**Comment by yvwbimpong** [comment]
https://github.com/user-attachments/assets/e1428849-59de-4668-b130-0e1d5fede947

**Comment by edulix** [comment]
Fixed some more related issues, also fixes #1505
